perception_dataset/deepen/deepen_to_t4_converter.py

```diff
@@ -287,6 +318,44 @@ def _format_deepen_annotation(
                     }
                 )
 
+            if label_dict["label_type"] == LabelType.SEGMENTATION_2D:
+                sensor_id = int(label_dict["sensor_id"].strip("sensor_"))
```

Comment:
Can we assume that `label_dict["sensor_id"]` is like `sensor_1`?

Reply:
This is my mistake, `int(label_dict["sensor_id"].strip("sensor"))` is more appropriate.

Reply:
In https://github.com/tier4/tier4_perception_dataset/pull/164/commits/7c449e6cdb30db3d332a0dda107267a1399c0ba1, I updated to strip `sensor` instead of `sensor_`.